scintillometry/io/psrfits/hdu.py

```diff
@@ -170,18 +251,34 @@ def start_time(self):
     def nrow(self):
         return self.header['NAXIS2']
 
+    @nrow.setter
+    def nrow(self, val):
```

Comment:
I'd prefer to just call `val` the name of what one is setting, i.e., `nrow`. If not, at least don't use an abbreviation, just `value`.